src/ui/src/core_components/content/CoreJsonViewer.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+	<div class="CoreJsonViewer">
+		<BaseJsonViewer v-if="data" :data="data" :paths-open="pathsOpen" />
+		<BaseJsonViewerChildrenCounter v-else :data="{}" />
+	</div>
+</template>
+
+<script lang="ts">
+import {
+	accentColor,
+	cssClasses,
+	secondaryTextColor,
+	separatorColor,
+} from "../../renderer/sharedStyleFields";
+import {
+	FieldCategory,
+	FieldType,
+	WriterComponentDefinition,
+} from "../../writerTypes";
+
+const description = "A component to explore JSON data as a hierarchy.";
+
+const initialData = {
+	name: "JSON Viewer",
+	description: "A JSON tree viewer where you can expand the keys.",
+	sample: {
+		description: "This sample is opened by default",
+		bool: true,
+		null: null,
+		list: [1, "two", { key: 3 }],
+	},
+	sampleClosed: {
+		description: "This sample is not opened by default",
+	},
+	createdAt: new Date(),
+};
+
+const definition: WriterComponentDefinition = {
+	name: "JSON Viewer",
+	description,
+	category: "Content",
+	fields: {
+		data: {
+			name: "Data",
+			init: JSON.stringify(initialData),
+			type: FieldType.Object,
+		},
+		pathsOpen: {
+			name: "Paths open",
```

Comment:
you should add `desc` attribute to explain it accepts a list of jsonpath expression :

```
list of JSONPATH expression. describe the path to opens
```